src/brackets/index.js

```diff
@@ -4,6 +4,26 @@
  * @param {string} str The string of brackets.
  * @returns {"valid" | "invalid"} Whether or not the string is valid.
  */
-function isValid(str) {}
+function isValid(str) {
+    let brackets = '()[]{}'
+    let stack = []
+
+    for (let bracket of str) {
+        let bracketIndex = brackets.indexOf(bracket)
+
+        if(bracketIndex === -1) {
+            continue
+        }
```

Comment:
What is the implication of this? Wouldn't that mean the string is not valid and you can return?

Reply:
yeah